src/main/resources/db/migration/V1_3__create_nav_utland.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE NAV_UTLAND
+(
+    id            SERIAL PRIMARY KEY,
+    uuid          VARCHAR(50) NOT NULL UNIQUE,
+    personident   VARCHAR(11) NOT NULL UNIQUE,
+    er_nav_utland BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Superpirk: er kan være is, siden er ikke er et domeneord. Men vet ikke helt om dette er noe vi har fulgt konsekvent i våre databaser.

Reply:
Har prøvd å se litt på formatet i våre andre databaser nå 🙌🏼  Vi har ikke så mange `boolean`-felt rundt omkring, og av de jeg fant har vi ingen med `is`/`er` eller `har`/`has` -prefix 🤔 
Det er også litt forskjellig rundt omkring mtp bruk av norske og engelske ord. For meg spiller det ingen rolle, så kan bytte til `is_nav_utland` siden det er vel den linjen vi kjører ellers 👍🏼